sdk/keyvault/keyvault-keys/types/src/core/clientOptions.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Similar comment here re: ./types directory. We should go ahead and remove this as it's created during the build step.

Reply:
Done! Also opened this PR based on our conversation: https://github.com/Azure/azure-sdk-for-js/pull/3322